### PR TITLE
Update eip155-8217.json

### PR DESCRIPTION
**Pull request by benjioh5**
- remove Klayn Official JSON-RPC route (Klaytn official API doesn't work any more)
- add ozys JSON-RPC route
- add klaytnscope uri

### CODE REVIEW

**Review by ligi — CHANGES_REQUESTED** [review]
This explorer violates EIP3091 when it comes to addresses

**Comment by benjioh5** [comment]
@ligi remove standard info

File: _data/chains/eip155-8217.json [hunk 1]
```diff
@@ -15,5 +15,9 @@
   "shortName": "Cypress",
   "chainId": 8217,
   "networkId": 8217,
-  "slip44": 8217
+  "slip44": 8217,
+  "explorers": [{
+    "name": "Klaytnscope",
+    "url": "https://scope.klaytn.com",
+  }]
```
Comment (ligi) [inline _data/chains/eip155-8217.json:22]:
```suggestion
  "explorers": [{
    "name": "Klaytnscope",
    "url": "https://scope.klaytn.com",
    "standard": "none"
  }]
```

**Comment by benjioh5** [comment]
Thanks a lot!

**Pull request merged**